src/asana/client.py

```diff
@@ -146,13 +171,29 @@ def create_task(project_id: str, due_date_str: str = None) -> str:
     return AsanaClient.singleton().create_task(project_id, due_date_str=due_date_str)
 
 
+def create_subtask(
+    parent_task_id: str, reviewer_id: str, task_name: str, task_description,
+    due_date_str: str = None
+) -> str:
+    """
+    Creates an Asana task and makes is a subtask of the given task id
+    """
+    return AsanaClient.singleton().create_subtask(
+        parent_task_id, reviewer_id, task_name, task_description, due_date_str
+    )
+
+
 def update_task(task_id: str, fields: dict):
     """
     Updates the specified Asana task, setting the provided fields
     """
     return AsanaClient.singleton().update_task(task_id, fields)
 
 
+def is_task_completed(task_id) -> bool:
```

Comment:
I think the pattern established here is to keep the naming consistent

```suggestion
def get_task_completed_status(task_id) -> bool:
```